src/components/landing/landing.js

```diff
@@ -19,115 +19,76 @@ class Landing extends Component {
     this.props.onSubmit();
   };
 
-  splitDataIntoBlocks = itemsArray => {
-    return itemsArray.reduce((acc, c, i) => {
-      if ((i && 3) === 0) acc.push([]);
-      acc[acc.length - 1].push(c);
-      return acc;
-    }, []);
-  };
-
-  renderCarouselBlock = eventBlock => (
-    // TODO: DELETED CLASS FOR
-    // <div className="carousel-block wd-100">
-    <div className="container">
-      {eventBlock.map(event => (
-        <EventsListItem key={event.id} event={event} />
-      ))}
-    </div>
-  );
-
   render() {
     const { items } = this.props.events.data;
-    const events = this.splitDataIntoBlocks(items);
-
-    // const carouselNavIsVisible = events.length > 1;
-
     return (
-      <>
-        <div className="main">
-          <HeadArticle />
-          <article className="works-article text-center">
-            <div className="works-title">
-              <h2>How EventsExpress Works</h2>
+      <div className="main">
+        <HeadArticle />
+        <article className="works-article text-center">
+          <div className="works-title">
```

Comment:
Extra div

Reply:
We don't use that in our design. It's all spelled out in the "About Us" tab. I delete that.